### PR TITLE
Added Elevate Pause Validation

### PR DESCRIPTION
**Pull request by voduyemi**
WI: [W-11565903](https://gus.lightning.force.com/a07EE000013aeXqYAI)
# Critical Changes

# Changes
- We've added validation for the Elevate pause functionality.

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata
**Custom Label**
- RD2_ElevatePauseInstallmentDateErrorMessage

# Deleted Metadata

### CODE REVIEW

**Review by ErinWiedemer — APPROVED** [review]
LGTM! Thanks!

**Review by npsp-reedestockton — APPROVED** [review]
Looks good, @voduyemi! Left one trivial comment.

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -1609,6 +1609,14 @@
         <shortDescription>Cannot change payment method to a non-Elevate payment method</shortDescription>
         <value>Valid Payment Methods for an active Elevate-connected Recurring Donation are Credit Card or ACH.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ElevatePauseInstallmentDateErrorMessage</fullName>
+        <categories>Recurring-Donations, Pause</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error message when trying to pause an RD a day from instalment date</shortDescription>
```
Comment (voduyemi) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:1617]:
```suggestion
        <shortDescription>Error message when trying to pause an RD a day from installment date</shortDescription>
```

**Review by daniel-fuller — APPROVED** [review]
looks good!

**Pull request closed without merging**